### PR TITLE
LPS-176818 | Test Automation | Master

### PR DESCRIPTION
**Pull request by davisaints**
- [LPS-176818](https://issues.liferay.com/browse/LPS-176818)

### CODE REVIEW

File: modules/apps/object/object-test/src/testFunctional/macros/ObjectAdmin.macro [hunk 1]
```diff
@@ -199,6 +199,14 @@ definition {
 			}
 		}
 
+		if (isSet(formulaFieldOption)) {
+			Click(locator1 = "ObjectField#SELECTED_FORMULA_FIELD");
+
+			Click(
+				key_dropdownItem = ${formulaFieldOption},
+				locator1 = "ObjectField#SELECT_FORMULA_FIELD_OPTION");
```
Comment (epark18) [inline modules/apps/object/object-test/src/testFunctional/macros/ObjectAdmin.macro:207]: Can you replace this using the Select function?

File: modules/apps/object/object-test/src/testFunctional/paths/ObjectField.path [hunk 2]
```diff
@@ -102,6 +102,16 @@
...
 	</tr>
+	<tr>
+		<td>SELECTED_FORMULA_FIELD</td>
+		<td>//div[@class = 'input-group']//div[contains(@class, 'item')]//input[@value='Formula']</td>
+		<td></td>
+	</tr>
+	<tr>
+		<td>SELECT_FORMULA_FIELD_OPTION</td>
+		<td>//*[contains(@class,'menu show')]//button[contains(@class,'dropdown-item')]//div[text()='${key_dropdownItem}']</td>
+		<td></td>
```
Comment (epark18) [inline modules/apps/object/object-test/src/testFunctional/paths/ObjectField.path:113]: if you can get the Select function to work you wont need this path

**Review by epark18 — COMMENTED** [review]
@davisaints I left a suggestion on using the Select function. Overall looks good

**Comment by davisaints** [comment]
ci:test:sf

File: modules/apps/object/object-test/src/testFunctional/tests/ObjectFields.testcase [hunk 3]
```diff
@@ -508,6 +508,33 @@ definition {
 		}
 	}
 
+	@description = "LPS-164948 - Verify that the user is able to add formula fields when managing System Objects"
+	@priority = 5
+	test CanCreateFormulaFieldOnSystemObject {
+		property portal.upstream = "true";
```
Comment (epark18) [inline modules/apps/object/object-test/src/testFunctional/tests/ObjectFields.testcase:514]: This property is defined in the testcase definition. I think you meant portal.acceptance

**Comment by epark18** [comment]
ci:forward

**Comment by davisaints** [comment]
ci:reopen

**Comment by davisaints** [comment]
ci:test:sf

**Comment by davisaints** [comment]
ci:forward

**Pull request closed without merging**